.github/workflows/build.yaml

```diff
@@ -9,11 +9,39 @@ on:
       - master
 
 jobs:
-  hello:
-    name: Hello
+  static-analysis:
+    name: Static Analysis
 
     runs-on: ubuntu-latest
 
+    strategy:
+      matrix:
+        php-version:
```

Comment:
By introducing a matrix strategy here, we can reference the resolved values using `${{ matrix.php-version }}`.

Since we only run the static analysis for a single PHP version, it might seem that it does not make a lot of sense.

However, in my experience, it's quite useful, as we can reference the `${{ matrix.php-version }}` in multiple places without having to hard-code the actual PHP version.

Reply:
Yeah, that makes sense. Perhaps it could be named just `php` for easier referencing of `matrix.php`.